Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -136,8 +136,8 @@ lemma smul_fin3_ext (P : Fin 3 → R) (u : R) :
 
 /-- The multiplicative action on a point representative. -/
 scoped instance instMulActionPoint : MulAction R <| Fin 3 → R where
-  one_smul _ := by simp only [smul_fin3, one_pow, one_mul, fin3_def]
-  mul_smul _ _ _ := by simp only [smul_fin3, mul_pow, mul_assoc, fin3_def_ext]
+  one_smul _ := by simp_rw [smul_fin3, one_pow, one_mul, fin3_def]
+  mul_smul _ _ _ := by simp_rw [smul_fin3, mul_pow, mul_assoc, fin3_def_ext]
```

Comment:
Surely `simp only` still works?

Reply:
Yeah, but `simp_rw` is shorter and allows you to see intermediate goal states, which is why I prefer it. I never tested if there are noticeable performance differences compared to `simp only`, and these two proofs are possibly too trivial for `simp_rw` to be beneficial?

Reply:
Makes sense, but I'm too used to `simp only`. One of these days I'll change all my `simp only` into `simp_rw`'s.